staff/lab/last-lab

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+import argparse
+import functools
+import sys
+
+import ocflib.lab.stats
+from ocflib.infra.hosts import hostname_from_domain
+from tabulate import tabulate
+
+
+def make_fqdn(s):
+    """Argparse custom type that appends ".ocf.berkeley.edu" to short hostnames."""
+    if '.' not in s:
+        return s + '.ocf.berkeley.edu'
+    else:
+        return s
+
+
+def build_query(args):
+    """Builds the MySQL query string with the given conditions."""
+    query = 'SELECT * FROM `session`'
```

Comment:
`TIMESTAMPDIFF(start, end, minute)/60 as duration` is also a useful field to have, IMHO

Reply:
I agree it could be useful. I'd prefer to implement it in `format_row` instead by subtracting `end` and `start` though. Topic for a future PR?